language/move-prover/interpreter/src/concrete/evaluator.rs

```diff
@@ -134,7 +134,7 @@ impl<'env> Evaluator<'env> {
                 if err == BigInt::zero() {
                     return;
                 }
-                self.record_evaluation_failure(exp, err);
+                self.record_evaluation_failure(exp, "Unexpected error code");
```

Comment:
```suggestion
                self.record_evaluation_failure(exp, "unexpected error code");
```

This would require an update to the `.exp` file as well.

Reply:
fixed